content/_guides/platform-behavior.md

```diff
@@ -41,5 +41,7 @@ You can check Component logs the same way as shown [here](managing-flow-errors),
 
 ![](/assets/img/integrator-guide/behavior/Screenshot_2.png)  
 
+In this case, the message can be lost. If you are aware that this scenario is possible in your case then you can set the memory limit to a custom value. Or ask to do this someone who owns components repository.
```

Comment:
In this case, the message can be lost. If you are aware that this scenario is possible in your case, then you can set memory limit to a custom value, more suitable for you. You can also ask Components repository owner to do this.